elytron/src/main/resources/schema/wildfly-elytron_community_18_1.xsd

```diff
@@ -0,0 +1,6450 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="urn:wildfly:elytron:community:18.1"
```

Comment:
Need to think about what is the correct updates needed here but we avoid minor bumps.